<launch>
	<arg name="sim" default="false"/>
	<arg name="lookahead" default=".15"/>
	<arg name="max_reacquire" default="3.0"/>
	<arg name="speed" default=".25"/>

	<group if="$(arg sim)">
		<arg name="base_frame" default="base_link"/>
		<node pkg="lab6" type="pure_pursuit.py" name="Trajectory_follower">
			<param name="odom_topic" value="$(arg base_frame)"/>		
			<param name="trajectory_topic" value="/trajectory/current"/>
			<param name="lookahead" value="$(arg lookahead)"/>
			<param name="speed" value="$(arg speed)"/>
			<param name="wheelbase" value="0.25"/>
			<param name="wrap" value="0"/>
			<param name="drive_topic" value="drive"/>
		</node>
	</group>


	<group unless="$(arg sim)">
		<arg name="base_frame" default="base_link"/>
		<node pkg="lab6" type="pure_pursuit.py" name="Trajectory_follower">
			<param name="odom_topic" value="$(arg base_frame)"/>		
			<param name="trajectory_topic" value="/trajectory/current"/>
			<param name="lookahead" value="$(arg lookahead)"/>
			<param name="speed" value="$(arg speed)"/>
			<param name="wheelbase" value="0.25"/>
			<param name="wrap" value="0"/>
			<param name="drive_topic" value="/vesc/high_level/ackermann_cmd_mux/input/nav_0"/>
		</node>
	</group>

	
	
</launch>
